charts/kubeclarity/values.yaml

```diff
@@ -296,33 +295,20 @@ kubeclarity-sbom-db:
 #######################################################################################
 
 #######################################################################################
-## KubeClarity Postgres Values
-
+## KubeClarity Internal Postgres Values
+## Use kubeclarity-postgresql if you want this chart to deploy a PostgreSQL instance
 kubeclarity-postgresql:
   enabled: true
 
-  ## Specify posgtresql image
-  # image:
-    # registry: docker.io
-    # repository: bitnami/postgresql
-    # tag: 14.4.0-debian-11-r4
-
-  ## initdb parameters
-  # initdb:
-    ##  ConfigMap with scripts to be run at first boot
-    ##  NOTE: This will override initdb.scripts
-    # scriptsConfigMap
-    ##  Secret with scripts to be run at first boot (in case it contains sensitive information)
-    ##  NOTE: This can work along initdbScripts or initdbScriptsConfigMap
-    # scriptsSecret:
-    ## Specify the PostgreSQL username and password to execute the initdb scripts
-    # user:
-    # password:
-
   ## Setup database name and password
   auth:
-    existingSecret: kubeclarity-postgresql-secret
+    user: postgres
```

Comment:
per https://artifacthub.io/packages/helm/bitnami/postgresql#postgresql-common-parameters field name is `username`

Reply:
@FrimIdan Updated